### DISCUSSION TITLE
Why use Microsoft Edge on Linux - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#21] [General]
# Why use Microsoft Edge on Linux - Alan Pope's blog

Yesterday, I wrote a little about the applications I’ve seen crash on my Ubuntu Linux laptop over the last six months.
Some people questioned why I use Microsoft Edge as my primary web browser on Ubuntu. I thought I’d write up why, and how a couple of the built-in features are appealing to me.
tl;dr it’s multiple profiles, stability, speed, tab sleep, and vertical tabs.
Multiple personality disorder I have tried to keep work and personal browser profiles separate for some years now.

https://popey.com/blog/2023/08/why-use-microsoft-edge-on-linux/

**Comment by aptanet** [comment]
Interesting read. I remember a period with Firefox and losing saved tabs, but always managed to recover them. Not had any of those problems for a long while now though. My personal experience has always been that Firefox is faster, more reliable and uses less resources than Chrom{e|ium} - counter what some others have found clearly. I may have to try Edge, but having been stuck with Windows on my laptop for rather too long now (a combination of lack of time to sort it and lack of space for the dual boot, which sadly I need) Edge irritates me with its insistence on trying to take over at every opportunity and trying to force various new features down my throat (and I've not wanted to spend any time working out how to counter that) - it has also become my Teams replacement for the times I have to suffer that given it only runs in Edge, Chrome, etc..

**Comment by makosol** [comment]
Reason not to use Edge : it's not free software.

**Reply by popey** [reply]
What Free Software browser should I use that has the features I desire?

**Comment by castrojo** [comment]
The built in chatGPT in the sidebar became really useful for me once I got over my self-loathing for using it.

**Comment by mercster** [comment]
I use Edge in Windows, but never considered it for Linux (which I've been running since the early 90s.)  Wasn't sure the Linux build was "mature."  I came across your article about vertical tabs and thought it sounded cool, so I looked up the Edge download for Linux.  As of 12/11/23 I can report the version number is at parity with the Windows build (the one that's available for download anyway.)

I can also report that, while vertical tabs are certainly interesting, it's too far a bridge for me to cross at this time.  And with 1440p it reduces video windows too much.  HOWEVER... it is much snappier than Chrome under Linux.  Comparatively, you can almost see Chrome thinking as it opens tabs, loads pages, etc (almost as dramatic as the speed difference between Firefox and Chrome.)  A very unscientific comparison of the two task managers shows Edge to be using a bit more RAM than Chrome.  But, ya know... RAM is cheap these days, and it's not drastic.  Exporting/importing bookmarks and passwords was a bit of a pain (gone are the days of telling the browser to "Just suck all the settings from XYZ browser!", at least in Linux) but... I will stick with it for awhile.  Some weird color palette issues aside, it's definitely promising.  And frankly, at this point, I like Microsoft more than Google (my younger self's head would burst into flames at this assertion.)

**Reply by MiqW** [reply]
If you're running any other Chromium-based browser in Linux, there's a really easy way to "import" bookmarks. Simply go into your profile -> Default, and locate the Bookmarks file. Open the same location in your new browser install, and just drag'n'drop the Bookmarks file across. Next time you fire-up your new browser, all your bookmarks are there, showing in the Bookmarks bar.......and set-up exactly the same way as they were in your other browser....

**Reply by mercster** [reply]
Yes, that's what I did.  I remember a time when browsers just kinda knew how to suck the data out of another browser... maybe the data's been put into something other than a text file that's easily slurpt up or something, I don't know.  With Export/import you also need to go to the trouble of moving things out of "Imported bookmarks" and replacing the bookmark bar links but, yeah I got it done.  Thanks.

EDIT:  ohh, yeah I didn't just replace a plaintext file, cuz I assumed there wasn't one (stored in some simple database or something, I dunno.)  I will find that bookmarks.html or whatever in the future and see if I can't just copy it across wholesale.  Cheers.

**Reply by MiqW** [reply]
My own Chrome history goes all the way back to the beta 'trial' builds in the summer of 2007. I signed-up for the test program; at that point, I'd been using FF for a few years. It was a real eye-opener when I originally switched from Internet Exploder in the early 2000s, but by the mid-to-late 2000s it was getting so it had horrendous memory leaks, and regularly crashed, taking all your work with it...

In the beginning, Chrome was a revelation. It was lightweight, sizzlingly fast, incredibly responsive and absolutely rock-solid in use. Fast-forward nearly 17 years and look at it now; chubby, bloated, ponderous.....and unrecognisable as the sprightly youngster it once was. Yet for all that, I'm glad Google had the foresight to open-source the thing via the Chromium Project. That code-base has been put to a LOT of good use in the intervening years.....in addition to which, it gets the benefit of thousands of pairs of eyes on the code world-wide.

Mike.

**Reply by mercster** [reply]
Yeah, every big "revolution" in browser tech feels like a sea change.  Imagine going from NCSA Mosaic to Netscape Navigator.  ;-)
